coralproject/talk
# [next] Asset Comment Counts

## What does this PR do?

Cache comment count data on the Asset.

## How do I test this PR?

Create a few comments, some containing banned words, and verify with the following graph query:

```graphql
{
  asset(id: "...") {
    id
    commentCounts {
      totalVisible
      statuses {
        ACCEPTED
        REJECTED
        PREMOD
        NONE
      	SYSTEM_WITHHELD
      }
    }
  }
}

```
